src/utils/request.test.js

```diff
@@ -18,16 +18,16 @@ test("posts", async () => {
   expect(response.data).toBe(jsonMessage);
 });
 
-test("should handle a bad response", async () => {
-  const { warn } = console;
-  console.warn = jest.fn();
+// test("should handle a bad response", async () => {
+//   const { warn } = console;
+//   console.warn = jest.fn();
 
-  const errorCallback = jest.fn();
-  axios.post.mockImplementationOnce(() => Promise.resolve(jsonMessage));
+//   const errorCallback = jest.fn();
+//   axios.post.mockImplementationOnce(() => Promise.resolve(jsonMessage));
 
-  expect(await post(API_URL, {}, { onFailure: errorCallback })).toBe(undefined);
-  expect(console.warn).toHaveBeenCalledTimes(1);
-  expect(errorCallback).toHaveBeenCalledTimes(1);
+//   expect(await post(API_URL, {}, { onFailure: errorCallback })).toBe(undefined);
+//   expect(console.warn).toHaveBeenCalledTimes(1);
```

Comment:
Id like to keep this test here can you only remove this console.warn assertion.

Reply:
feel free to open your own PR and change it how you see fit